caravel/templates/listing_form.html

```diff
@@ -17,12 +17,11 @@
           {% if form.principal %}
             {{ wtf.form_field(form.principal) }}
           {% endif %}
-					<div class="alert alert-warning" role="alert">
-          	BSD/Medicine/Medical School affiliates: if you are unable to sign
-            in with your CNetID, please enter your email in the box. Your
-            listing might not be posted immediately. We reserve the right to
-            remove listings at any time.
-					</div>
+          <p class="alert alert-warning" role="alert">
```

Comment:
I think it is. The `<p>` is more semantic than a `<div>`, but it should show up in the same manner.